b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/role/controllerEditRolesToAddOrRemoveUser/controllerDecodeEditRolesToAddOrRemoveUser.ts

```diff
@@ -28,22 +29,35 @@ function getRolesThatNeedToRemoveUser(initRoleList: string[], roleList: string[]
     return initRoleList.filter(roleUri => !roleList.includes(roleUri));
 }
 
+async function patchRoleDetails(session: Session, userId: string, rolesUriList: string[], patchMethod: PatchMethod)
+    : Promise<boolean> {
+
+    for (const uri in rolesUriList) {
+        const role: Role | null = await controllerDecodePatchRole(session, uri, patchMethod, "users", [ userId ]);
+
+        if (!role) {
+
```

Comment:
unnecessary new lines. check other places also.

Reply:
Space is required before a `return`.